src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--month-length--29 calendar--start-day--fri  calendar--pos">
```

Comment:
you only can use modificator (I mean 'calendar--pos) to position elements inside your block, not to position your block

Reply:
Thank you for suggestions and remarks! All of them make sense. I'll fix fix my issues.